scripts/variantstore/docs/aou/AOU_DELIVERABLES.md

```diff
@@ -50,41 +45,55 @@
-13. `GvsCallsetCost`
-    - The cost from this callset, which represents the total BigQuery cost (which is not represented in the Terra UI total workflow cost) from the GVS pipeline workflows, is used to calculate the cost of the callset as a whole and by sample.
-
+    - This workflow extracts the control data in BigQuery and transforms it into a sharded joint VCF incorporating the VQSR filter set data.
+    - It will need to be run once with `control_samples` set to "true", and with the `filter_set_name` from the `GvsCreateFilterSet` step and the `extract_table_prefix` from the `GvsPrepareRangesCallset` step.  
+    - Include a valid `output_gcs_dir` parameter, which is where the VCF, interval list, manifest, and sample name list files will go, e.g. `gs://fc-<workspace-id>/extractedcontrol-vcfs/`
+    - This workflow does not use the Terra Data Entity Model to run, so be sure to select the `Run workflow with inputs defined by file paths` workflow submission option.
+10. `GvsCallsetStatistics` workflow
+    - You will need to have the "BigQuery Data Viewer" role for your @pmi-ops proxy group on the `spec-ops-aou:gvs_public_reference_data.gnomad_v3_sites` table
+    - This workflow needs to be run with the `extract_table_prefix` input from `GvsPrepareRangesCallset` step.
+    - This workflow needs to be run with the `filter_set_name` input from `GvsCreateFilterSet` step.
+    - This workflow does not use the Terra Data Entity Model to run, so be sure to select the `Run workflow with inputs defined by file paths` workflow submission option.
+11. `GvsCalculatePrecisionAndSensitivity` workflow
+    - You will need to have "Storage Object View" access granted for your @pmi-ops proxy group on the `gs://broad-dsp-spec-ops/gvs/truth` directory
+    - This workflow needs to be run with the control sample chr20 vcfs from `GvsExtractCallset` step which were placed in the `output_gcs_dir`.
+    - This workflow does not use the Terra Data Entity Model to run, so be sure to select the `Run workflow with inputs defined by file paths` workflow submission option.
+12. `GvsCallsetCost` workflow
+    - This workflow calculates the total BigQuery cost of generating this callset (which is not represented in the Terra UI total workflow cost) using the above GVS workflows; it's used to calculate the cost as a whole and by sample.
 
 ## Deliverables (via email to stakeholders once the above steps are complete)
-1. GCS locations of the VCFs, indexes and interval_list files (subpaths of the `output_gcs_dir` input from `GvsExtractCallset`)
+1. GCS location of the VDS (the `--vds-path` input from `hail_gvs_import.py`)
 2. fully qualified name of the BigQuery dataset (composed of the `project_id` and `dataset_name` inputs from the workflows)
-3. callset statistics CSV file (see step #11 above)
-4. TSV output from `GvsCalculatePrecisionAndSensitivity` workflow (see step #12 above)
+3. CSV output from `GvsCallsetStatistics` workflow
```

Comment:
Probably a good idea to link to vds/delivery/Delivering%20a%20VDS.md since that includes details of how to get the VDS to the AoU delivery bucket, naming conventions...etc.

Reply:
great idea